classes/User.php

```diff
@@ -1254,6 +1273,66 @@ public function getAvatarSize(): int {
         return $this->forbidden_seal ? self::AVATAR_MAX_SEAL_SIZE : self::AVATAR_MAX_SIZE;
     }
 
+    public function canChangeChatColor(): bool {
+        // Premium purchased
+        if($this->premium_credits_purchased) {
+            return true;
+        }
+
+        // Forbidden seal
+        if($this->forbidden_seal && $this->forbidden_seal['time'] > time()) {
+            return true;
+        }
+
+        // Staff level
+        if($this->isModerator() || $this->isHeadModerator() || $this->isContentAdmin() || $this->isUserAdmin() || $this->isHeadAdmin()) {
+            return true;
+        }
+
+        return false;
+    }
+
+    /**
+     * @param false $forceAll // should only be used for debugging
+     * @return string[]
+     */
+    public function getNameColors($forceAll = false) {
```

Comment:
Head admin should be able to use all colors, is this debug code worth pushing to production?